modules/admin/proc-configuring-postgresql-instance-using-helm.adoc

```diff
@@ -0,0 +1,132 @@
+[id="proc-configuring-postgresql-instance-using-helm_{context}"]
+= Configuring an external PostgreSQL instance using the Helm Chart
+
+You can configure an external PostgreSQL instance by using the Helm Chart. By default, the Helm Chart creates and manages a local instance of PostgreSQL in the same namespace where you have deployed the {product-very-short} instance. However, you can change this default setting to configure an external PostgreSQL database server, for example, AWS RDS or Azure database. 
+
+.Prerequisites
+
+* You have the following details:
+** `db-host`: Denotes your PostgreSQL instance Domain Name System (DNS) or IP address 
+** `db-port`: Denotes your PostgreSQL instance port number, usually `5432`
+** `username`: Denotes the user name to connect to your PostgreSQL instance
+** `password`: Denotes the password to connect to your PostgreSQL instance
+
+* You have a PostgreSQL certificate of the .pem format so that you can secure your database connection using the Secure Socket Layer (SSL) or Transport Layer Security (TLS) protocol. For more information, see link:{LinkAdminGuide}#postgresql-certificate-for-aws-rds_admin-rhdh[Downloading PostgreSQL certificate for AWS RDS] or link:{LinkAdminGuide}#generating-postgreSQL-certificate-for-azure-database_admin-rhdh[Generating PostgreSQL certificate for Azure database].
+
+[NOTE]
+====
+By default, {product-short} uses a database for each plugin and automatically creates it if none is found. As an administrator, know that users might need the `Create Database` privilege in addition to `PSQL Database` privileges for configuring an external PostgreSQL instance.  
+====
+ 
+
+.Procedure
+
+. Create a credential secret to connect with the PostgreSQL instance:
++
+[source,yaml]
+----
+cat <<EOF | oc -n <your-namespace> create -f -
+apiVersion: v1
+kind: Secret
+metadata:
+ name: <cred-secret> <1>
+type: Opaque
+stringData: <2>
+ POSTGRES_PASSWORD: <password>
+ POSTGRES_PORT: "<db-port>"
+ POSTGRES_USER: <username>
+ POSTGRES_HOST: <db-host>
+ PGSSLMODE: <ssl-mode> # for TLS connection <3>
+ NODE_EXTRA_CA_CERTS: <abs-path-to-pem-file> # for TLS connection, e.g. /opt/app-root/src/postgres-crt.pem <4>
+EOF
+----
+<1> Provide the name of the credential secret.
+<2> Provide credential data to connect with your PostgreSQL instance.
+<3> Optional: Provide the value only if you need a TLS connection for your PostgreSQL instance.
+<4> Optional: Provide the value only if you need a TLS connection for your PostgreSQL instance.
```

Comment:
I see your point, the sections (3.5 and 3.7) created for TLS connection is specific to the Crunchy database. On the other hand, the ones (section 3.4 and 3.6) without TLS connection is specific to AWS RDS and Azure database. The sections  (3.5 and 3.7)  specific to TLS connection was created as per the scope of this JIRAhttps://issues.redhat.com/browse/RHIDP-1238.

Reply:
Okay, I had no idea that 3.2 and 3.4 are specific for Crunchy database. Tweaking the headings would definitely help to distinguish.

All the supported databases (Crunchy, AWS RDS and Azure database) can use TLS connection, although they might not be enforced.

Reply:
I do not think we should make our headings vendor-specific. The segregation point i see is the presence of a TLS private key. So, i have modified the headings accordingly.  

- Configuring an external PostgreSQL instance using the Operator
- Configuring an external PostgreSQL instance with a TLS private key using the Operator
- Configuring an external PostgreSQL instance using the Helm Chart
- Configuring an external PostgreSQL instance with a TLS private key using the Helm Chart

Reply:
In my eyes, it makes sense to have just two guides -  for Operator and Helm. 
They will contain the basic configuration without TLS as required, the steps for configuring TLS as optional, and the same for using a TLS private key.

- To use external Postgresql, configure this
- If you want to use TLS connection, configure this and use these values
- if your TLS connection require private key, configure this  and use these values

Reply:
Thaks @zdrapela for this suggestion ^^^. This would also help us to remove load of redundant content and present the information in more succinct manner.